Directory.Build.props

```diff
@@ -17,6 +17,7 @@
     <_hostOS Condition="$([MSBuild]::IsOSPlatform('ILLUMOS'))">illumos</_hostOS>
     <_hostOS Condition="$([MSBuild]::IsOSPlatform('SOLARIS'))">Solaris</_hostOS>
     <_hostOS Condition="$([MSBuild]::IsOSPlatform('WINDOWS'))">windows</_hostOS>
+    <HostOS>$(_hostOS)</HostOS>
```

Comment:
I think it'd be cleaner to set `HostOS` directly rather than going through the `_hostOS` indirection.

Reply:
Yup, it would be cleaner but we would still need a new property for that kind of a cleanup, which can be done separately.